src/client/pages/store-management/store-management-view.scss

```diff
@@ -0,0 +1,26 @@
+.store-management-view {
```

Comment:
I forgot to tell you that we need to keep that curly braces in new line in every where

Reply:
okay